module/zfs/vdev_indirect.c

```diff
@@ -1479,12 +1479,12 @@ vdev_indirect_all_checksum_errors(zio_t *zio)
 
 			vdev_t *vd = ic->ic_vdev;
 
-			(void) zfs_ereport_post_checksum(zio->io_spa, vd,
-			    NULL, zio, is->is_target_offset, is->is_size,
-			    NULL, NULL, NULL);
 			mutex_enter(&vd->vdev_stat_lock);
```

Comment:
I feel like this is a candidate for an atomic increment so that we can avoid more expensive locking operations. That can be left for future work.